### DISCUSSION TITLE
Android/iPhone App Recommendations

### DISCUSSION THREAD

**Discussion by intrinseca** [blakeblackshear/frigate#943] [Ask A Question] [answered]
Any recommendations for Android or iOS apps to view Frigate's proxied RTMP streams? I use tinyCam to connect directly to the cameras but would like to bounce this off of Frigate first.

**Comment by blakeblackshear** [comment, marked as answer]
Tinycam should be able to connect to frigate's rtmp relay too.

**Reply by intrinseca** [reply]
So it can! I didn't see RTMP in the protocols list and assumed it wouldn't work, but if you give it the RTMP URL and tell it to guess the protocol it works!

Do you have suggestions for iOS equivalents?

**Comment by mitchross** [comment]
Whats the need for an app when you can use the web UI ?

**Reply by Minglarn** [reply]
No need, but the Frigate WebUI is not protected. Would be nice if there was an user & pass protection if accessed from the www.

**Reply by mitchross** [reply]
@Minglarn check out my article. https://mitchross09.medium.com/how-i-self-host-my-own-sites-and-applications-with-unraid-docker-authelia-and-cloudflare-1f0d8e6f8912
